src/FSharpPlus/Builders.fs

```diff
@@ -83,7 +87,9 @@ module Builders =
             Using.Invoke (p.GetEnumerator () :> IDisposable) (fun enum ->
                 let enum = enum :?> IEnumerator<_>
                 this.While (enum.MoveNext, fun () -> rest enum.Current) : '``MonadPlus<'U>``)
-    
+
+    /// Strict (non lazy) monadic builder. Use when you to avoid potential side-effects due to additive behavior of monad plus.
```

Comment:
Typo: "when you **want** to"

Reply:
This sentence "Use when you to avoid potential side-effects due to additive behavior of monad plus." is wrong.
Actually it's exactly the opposite, this is when you want to have side effects, instead of additive behavior.